src/components/css/SearchBar.css

```diff
@@ -0,0 +1,15 @@
+.SearchBar--suggestion{
+    width: 100%;
+    background-color:rgba(231, 231, 231, 0.671);
+
+}
+
+.react-autosuggest__input{
+    width: 100%;
+}
```

Comment:
Fix indentation and spacings.

Weird naming convention, what do you use ? BEM ?